autoware_launch/launch/planning_simulator.launch.xml

```diff
@@ -96,4 +96,9 @@
       <arg name="vehicle_info_param_file" value="$(find-pkg-share $(var vehicle_model)_description)/config/vehicle_info.param.yaml"/>
     </include>
   </group>
+
+  <!-- Dummy doors -->
+  <group>
+    <include file="$(find-pkg-share vehicle_door_simulator)/launch/vehicle_door_simulator.launch.xml"/>
+  </group>
```

Comment:
It would be better to launch it in the tier4_simulator_component.launch.xml.

Reply:
Moved and add argument https://github.com/autowarefoundation/autoware_launch/pull/921/commits/29376b027cc2af60fffe82ad33eaa642e35df19e.